libbpf-cargo/src/btf/btf.rs

```diff
@@ -227,6 +227,50 @@ impl<'a> Btf<'a> {
         })
     }
 
+    pub fn type_default(&self, type_id: u32) -> Result<String> {
```

Comment:
Can you leave a doc comment here too? Maybe something along the lines of "Returns an expression that evaluates to the default value of type `typeid` in string form"